airflow/models/dag.py

```diff
@@ -2743,7 +2743,9 @@ def __init__(self, concurrency=None, **kwargs):
             else:
                 self.max_active_tasks = conf.getint('core', 'max_active_tasks_per_dag')
         if self.max_active_runs is None:
-            self.max_active_runs = conf.getint('core', 'max_active_runs_per_dag')
+            self.max_active_runs = kwargs.get(
+                'max_active_runs', conf.getint('core', 'max_active_runs_per_dag', fallback=16)
+            )
```

Comment:
Wait, why do we need the fallback? It should get it from the default config anyways.

Reply:
I just added it to be safe in case max_active_runs_per_dag is removed from the configuration. 
Can't think of any other reason why they'd get that errror

Reply:
Gotcha. They'd have to remove it from the `config_templates/default_airflow.cfg` in their install, so I think that is unlikely to be the cause.